### PR TITLE
Result Streaming settings

### PR DESCRIPTION
**Pull request by anthonydresser**
Adds a setting to control result set streaming

### CODE REVIEW

File: src/sql/parts/query/editor/gridPanel.ts [hunk 1]
```diff
@@ -188,45 +188,67 @@ export class GridPanel extends ViewletPanel {
...
+				} else {
+					warn('Got result set update request for non-existant table');
+				}
 			}
-		}
 
-		this.maximumBodySize = this.tables.reduce((p, c) => {
-			return p + c.maximumSize;
-		}, 0);
+			this.maximumBodySize = this.tables.reduce((p, c) => {
```
Comment (ranasaria) [inline src/sql/parts/query/editor/gridPanel.ts]: Is this change related to creating a setting for the results streaming feature? If not can you put comments in the pull request on the purpose of this change.

File: src/sql/parts/query/editor/gridPanel.ts [hunk 2]
```diff
@@ -188,45 +188,67 @@ export class GridPanel extends ViewletPanel {
...
+				return p + c.maximumSize;
+			}, 0);
+
+			if (this.state && this.state.scrollPosition) {
```
Comment (ranasaria) [inline src/sql/parts/query/editor/gridPanel.ts]: Is this change related to creating a setting for the results streaming feature? If not can you put comments in the pull request on the purpose of this change.

Comment (anthonydresser) [reply src/sql/parts/query/editor/gridPanel.ts]: this change specifically is just tabbing inwards, git is doing some weird things with diff. the only code change is the `let change = false` part which is just doing a check to update the ui or not.

**Comment by kburtram** [comment]
@anthonydresser I'll merge now to get included in the Dec release.  You can look at @ranasaria suggestion for a separate PR if necessary.

**Pull request merged**